numba/tests/test_parfors.py

```diff
@@ -525,13 +634,37 @@ def test_impl(A):
         self.check(test_impl, A)
         A = np.random.randint(10, size=n).astype(np.int32)
         self.check(test_impl, A)
+
         # test checking the number of arguments for the reduce function
         def test_impl():
             g = lambda x: x ** 2
             return reduce(g, np.array([1, 2, 3, 4, 5]), 2)
         with self.assertTypingError():
             self.check(test_impl)
 
+        # test checking reduction over bitarray masked arrays
```

Comment:
Given these tests, I think I correctly spotted masked reductions in the source above. Please could this feature be documented?

Reply:
Unfortunately, masked reductions is not ready yet. I had to turn it off in many cases until the infrastructure is improved.